spark-jobs/src/main/scala/filodb/downsampler/chunk/DownsamplerSettings.scala

```diff
@@ -80,13 +82,39 @@ class DownsamplerSettings(conf: Config = ConfigFactory.empty()) extends Serializ
 
   @transient lazy val exportRuleKey = downsamplerConfig.as[Seq[String]]("data-export.key-labels")
 
-  @transient lazy val exportBucket = downsamplerConfig.as[String]("data-export.bucket")
-
   @transient lazy val exportDropLabels = downsamplerConfig.as[Seq[String]]("data-export.drop-labels")
 
   @transient lazy val exportKeyToRules = {
     val keyRulesPairs = downsamplerConfig.as[Seq[Config]]("data-export.groups").map { group =>
       val key = group.as[Seq[String]]("key")
+      val tableName = group.as[String]("table")
+      val tablePath = group.as[String]("table-path")
+      val labelColumnMapping = group.as[Seq[String]]("label-column-mapping")
+        .sliding(2, 2).map(seq => (seq.head, seq.last)).toSeq
```

Comment:
for. my understanding, what does sliding (2, 2 ) does?

Reply:
Added comments to explain this sliding and to creation of labelColumnMapping Seq[(a,b), (c,d)]